app/controllers/TaskListController.scala

```diff
@@ -159,14 +160,17 @@ object TaskListController {
     schemeName: String,
     prefix: String,
     userAnswers: UserAnswers
-  ): TaskListItemViewModel =
+  ): TaskListItemViewModel = {
+    val taskListStatus: TaskListStatus = landOrPropertyInterestStatus(srn, userAnswers)
+
     TaskListItemViewModel(
       LinkMessage(
         Message(s"$prefix.interest.title", schemeName),
-        controllers.routes.UnauthorisedController.onPageLoad.url
-      ).checkQuestionLock(srn, userAnswers),
-      NotStarted
+        controllers.landorproperty.routes.LandOrPropertyContributionsController.onPageLoad(srn, NormalMode).url
+      ),
```

Comment:
`.checkQuestionLock(srn, userAnswers)` is required here. All questions on from member details (2) can only be started once the first two are complete.

Reply:
There are some changes I need to address with regards to the task status and alternative messaging when a question is locked. If you put the lock back I will address the status/message as per the prototype and conversation with @vasanthavijaya